variants/README.md

```diff
@@ -73,7 +73,7 @@ User data will be read from a mounted CD-ROM, either from a file named "user-dat
 
 #### aws-k8s-1.15: Kubernetes 1.15 node
 
-The [aws-k8s-1.15](aws-k8s-1.15/Cargo.toml) variant includes the packages needed to run a Kubernetes node in AWS.
+The aws-k8s-1.15 variant includes the packages needed to run a Kubernetes node in AWS.
 It supports self-hosted clusters and clusters managed by [EKS](https://aws.amazon.com/eks/).
 
 This variant is compatible with Kubernetes 1.15, 1.16, and 1.17 clusters.
```

Comment:
Why not just delete this section?

Reply:
This is under the `### Deprecated variants` section of the README. I thought it's meant to keep historical records of deprecated variants.

Reply:
At least for some period of time, I thought it'd be good to keep the listing here, in the deprecated section.  Users running 1.15 who suddenly can't find any mention of their product anywhere could be understandably peeved.  :)

Reply:
Should we change it to past tense, since it's gone from the tree?